src/qt/guiutil.cpp

```diff
@@ -1064,7 +1065,36 @@ void loadStyleSheet(QWidget* widget, bool fDebugWidget)
                 if (!qFile.open(QFile::ReadOnly)) {
                     throw std::runtime_error(strprintf("%s: Failed to open file: %s", __func__, file.toStdString()));
                 }
-                stylesheet->append(QLatin1String(qFile.readAll()));
+
+                QString strStyle = QLatin1String(qFile.readAll());
+                // Process all <os=...></os> groups in the stylesheet first
+                QRegularExpressionMatch osStyleMatch;
+                QRegularExpression osStyleExp("^(<os=(?:'|\").+(?:'|\")>)((?:.|\n)+?)(</os>?)$");
```

Comment:
nit: regular expressions are so easy to read! (no :D ) maybe smth like 79bd7ceb00?

Reply:
sure why not :D picked it!